.github/workflows/daily.yml

```diff
@@ -55,6 +58,10 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: "3.9"
+          cache: 'pip'
+          cache-dependency-path: |
+            requirements-tests.txt
```

Comment:
Should this contain `METADATA.toml` files? 🤔

Reply:
When we'll be able to specify non-types dependencies, are they going to be specified in `METADATA.toml` ? If so, then yes `cache-dependency-path` should contain `METADATA.toml` for stubtest jobs.

Reply:
> When we'll be able to specify non-types dependencies, are they going to be specified in `METADATA.toml` ?

I think that's the idea, yup!